libs/components/molecules/src/lib/footer/footer.component.html

```diff
@@ -0,0 +1,72 @@
+<!-- need to chnage the links -->
+<div class="footer">
+  <div class="container">
+    <form action="http://localhost:4400/?path=/story/footercomponent--primary" method="get" target="_blank">
+      <button class="button" mat-stroked-button>
+        <mat-icon>info</mat-icon>
+        Ilustration
+      </button>
+    </form>
+
+    <div class="dropup">
+      <button class="button" mat-stroked-button><mat-icon>file_download</mat-icon>Download</button>
+      <div class="dropup-content">
```

Comment:
Use material overlay as we discussed https://material.angular.io/cdk/overlay/overview